modules/moa-accessing-your-cluster.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/creating-first-moa-cluster.adoc
+
+
+[id="moa-accessing-your-cluster"]
+= Accessing your cluster
+
+To login to your cluster, you must configure an Identity Provider (IDP).
+
+This guide uses Github as an example IDP. 
+
+For other supported IDPs, run `moactl create idp --help`, and consult the OpenShift documentation on [configuring an IDP](https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#supported-identity-providers) for more information.
+
+.Procedure
+
+. Add an IDP.
+.. Run the following command to create an IDP backed by Github. Follow the prompts from the output to access your [Github developer settings](https://github.com/settings/developers) and configure a new OAuth application.
++
+[source, bash]
+----
+$ moactl create idp --cluster rh-moa-test-cluster1 --type github                                                                                               
```

Comment:
In our updated guidelines for the CLI, we're splitting the command and the output into two distinct blocks...

> The previous command returns the following output:

Reply:
I'll try this throughout.